ios/brave-ios/App/Client.xcodeproj/xcshareddata/xcschemes/Debug.xcscheme

```diff
@@ -100,6 +100,10 @@
             argument = "-com.apple.CoreData.ConcurrencyDebug 1"
             isEnabled = "YES">
          </CommandLineArgument>
+         <CommandLineArgument
+            argument = "-BREnableScreenTimeByDefault NO"
+            isEnabled = "NO">
```

Comment:
Disabled by default in all debug schemes. Devs can duplicate the scheme(s) (making sure `Shared` is unchecked) and enable if they wish to use this DebugFlag.